package-lock.json

```diff
@@ -9,8 +9,8 @@
             "resolved": "https://registry.npmjs.org/@types/bunyan/-/bunyan-1.8.4.tgz",
             "integrity": "sha1-acEa3HtQU41F+2jZrjnQYrlDLzg=",
             "requires": {
-                "@types/events": "1.1.0",
-                "@types/node": "6.0.96"
+                "@types/events": "*",
```

Comment:
I'm not sure why this PR includes changes to `package-lock.json`.  We're not changing any dependencies or version info in `package.json` so we shouldn't be revving versions in the lock file.  Can you back these out please?